### PR TITLE
chore(openchallenges): 2024-01-11 DB update

### PR DESCRIPTION
**Pull request by vpchung**
## Changelog

* Added:
   * 2 platforms: Devpost, Zindi
   * 10 challenges
      *  7 current & upcoming challenges from Devpost, categorized under "Health"
      * 3 ongoing challenges from Zindi, categorized under "Biology", "Health", or "Research"
   * 11 organizations
* Updated:
   * challenge plot numbers (+ 2024 numbers)

## Preview

* **Landscape**
Screenshot 2024-01-11 at 3 34 17 PM

* **Challenges plot**
Screenshot 2024-01-12 at 10 53 36 AM

* **Platforms list**
Screenshot 2024-01-11 at 3 34 28 PM

* **New challenges**
Screenshot 2024-01-11 at 3 34 46 PM

### CODE REVIEW

**Review by gaiaandreoletti — APPROVED** [review]

**Comment by vpchung** [comment]
@rrchai because I made some edits to the home page, I will need your approval too 😬  Whenever you have time - thanks!

File: apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/service/ChallengeAnalyticsService.java [hunk 1]
```diff
@@ -14,9 +14,9 @@ public ChallengesPerYearDto getChallengesPerYear() {
     List<String> years =
         Arrays.asList(
             "2007", "2008", "2009", "2010", "2011", "2012", "2013", "2014", "2015", "2016", "2017",
-            "2018", "2019", "2020", "2021", "2022", "2023");
+            "2018", "2019", "2020", "2021", "2022", "2023", "2024");
     List<Integer> challengeCounts =
-        Arrays.asList(6, 9, 13, 17, 23, 29, 34, 41, 49, 59, 86, 97, 116, 135, 183, 242, 302);
+        Arrays.asList(6, 9, 13, 17, 23, 29, 34, 41, 49, 59, 86, 97, 116, 135, 184, 243, 305, 315);
```
Comment (rrchai) [inline apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/service/ChallengeAnalyticsService.java:19]: It looks like total 13 challenges were added (or dates were defined): 1 in 2021, 2 in 2023, 10 in 2024. Is it correct?

Comment (vpchung) [reply apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/service/ChallengeAnalyticsService.java:19]: In this PR, only 7 were added (3 do not have dates). The extraneous counts are probably from me forgetting to update the plot in the previous DB updates.

**Review by rrchai — APPROVED** [review]
tested timeline plot and approved

**Pull request merged**